remoteappmanager/templates/home.html

```diff
@@ -1,39 +1,246 @@
-                  </button>
-                </span>
+{% extends "base.html" %}
+
+{% block body %}
+    <noscript>
+        <div id='noscript'>
+            Jupyter Hub requires JavaScript.<br>
+            Please enable it to proceed.
+        </div>
+    </noscript>
+
+    <div class="wrapper">
+
+        <!-- Main Header -->
+        <header class="main-header">
+
+            <a href="#" class="logo">
+                <!-- logo for regular state and mobile devices -->
+                <span class="logo-lg"><b>Simphony</b>Remote</span>
+            </a>
+
+        <!-- Header Navbar -->
+        <nav class="navbar navbar-static-top" role="navigation">
+            <!-- Navbar Right Menu -->
+            <div class="navbar-custom-menu">
+                <ul class="nav navbar-nav">
+
+                    <!-- User Account Menu -->
+                    <li class="dropdown user user-menu">
+                        <!-- Menu Toggle Button -->
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                            <!-- The user image in the navbar-->
+                            <img src="https://www.gravatar.com/avatar/{{ user.name | gravatar_id }}" class="user-image" alt="User Image">
+                            <!-- hidden-xs hides the username on small devices so only the image appears. -->
+                            <span class="hidden-xs">{{ user.name }}</span>
+                        </a>
+                        <ul class="dropdown-menu">
+                            <!-- The user image in the menu -->
+                            <li class="user-header">
+                                <img src="https://www.gravatar.com/avatar/{{ user.name | gravatar_id }}" class="img-circle" alt="User Image">
+                                <p>{{ user.name }}</p>
+                            </li>
+                            <!-- Menu Body -->
+                            <li class="user-body"></li>
+
+                            <!-- Menu Footer-->
+                            <li class="user-footer">
+                                <div class="pull-right">
+                                    {% block login_widget %}
+                                        <a id="logout" href="{{ logout_url }}" class="btn btn-default btn-flat">
+                                            <i class="fa fa-sign-out"> Sign out</i>
+                                        </a>
+                                    {% endblock %}
+                                </div>
+                            </li>
+                        </ul>
+                    </li>
+                    <!-- Control Sidebar Toggle Button -->
+                </ul>
+            </div>
+        </nav>
+        </header>
+
+        <div id="ember_container"></div>
+
+        {% raw %}
+        <!-- Ember template for the Application list -->
+        <script type="text/x-handlebars" data-template-name="application">
```

Comment:
The main Ember application which encapsulate the sidebar containing the application_list_component associated template and the main content containing the application_view_component associated template